src/js/view/reducers/logViewerReducer.js

```diff
@@ -27,12 +29,28 @@ export const logViewerReducer = (state = initialState, action) => {
     case 'LOGVIEWER_SET_LOG': {
       console.log('SETTING');
       const { sourcePath, log, startByteOfLines } = action.data;
+      let meanByteValueOfInitialLines = 0;
+      log.forEach(line => {
+        meanByteValueOfInitialLines = meanByteValueOfInitialLines + line.length;
+      });
+      meanByteValueOfInitialLines = Math.round(
+        meanByteValueOfInitialLines / log.length
+      );
+
```

Comment:
Realized this was repeating itself at a couple of places in the reducer. Making it a separate function and basing the calculation on the start byte data instead.
```
const calculateMeanValueOfBytesPerLine = startBytes => {
  return Math.round(
    (startBytes[startBytes.length - 1] - startBytes[0]) / startBytes.length
  );
};
```